.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -1,70 +1,5 @@
 jobs:
-- job: linux_64
```

Comment:
This PR shouldn't be changing files outside the recipe directory.

Reply:
I rolled these changes back.